Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs

```diff
@@ -903,12 +884,6 @@ public void DeleteTestFacilitiesFromDataset(string testDatasetIdentifier)
                     where: x => x.TestDatasetIdentifier == testDatasetIdentifier && !x.Deleted);
             }
         }
-
-        private static TimeSpan? ValidFromBeforeStartDate(Randomizer random)
-        {
-            return random.Int(1, 5) == 1 ? TimeSpan.FromDays(random.Int(1, 5)) : default(TimeSpan?); // set on 20% of opportunities
-        }
-
         private static TimeSpan? ValidFromBeforeStartDate(bool? validFromBeforeStartDate, DateTime startDate)
```

Comment:
Perhaps worth some comments in here explaining the resulting behavior?

Reply:
Also is there a risk that this logic creates a negative validFromBeforeStartDate, if the event is happening in the next 2 hours?

Reply:
I think this is handled by:
```
case false when startDate.Date == now.Date:
                    return TimeSpan.Zero;
```

Unless I've missed something.

Reply:
Changed it to return one hour.